### ISSUE TITLE
🐛onProcessTerminated Event Does not Reach on Global Subscriptions

### ISSUE THREAD

**Issue by alexanderkasten** [process-engine/process_engine_runtime#346]
## Description

The global notification which gets published if a process was terminated does not reach the subscriptions. 

## Reproduction

Try to use:

```
      this._managementApiService.onProcessTerminated(this.activeSolutionEntry.identity, (message: any) => {
        console.log(message);
      }, true);
```

**Comment by ElRaptorus** [comment]
It appears that Process Termination breaks the ProcessInstance's Promise-Chain, before the event is being published. 

This will require some more digging before I can say with certainty where or why this happens.

**Comment by alexanderkasten** [comment]
Additional Information:
If you terminate a process the `onProcessError` notification gets triggered.

@ElRaptorus

**Comment by ElRaptorus** [comment]
> Additional Information:
> If you terminate a process the `onProcessError` notification gets triggered.
> 
> @ElRaptorus

Curious.
Does this happen for `TerminateEndEvents`, or when you terminate a process manually?

**Comment by alexanderkasten** [comment]
> > Additional Information:
> > If you terminate a process the `onProcessError` notification gets triggered.
> > @ElRaptorus
> 
> Curious.
> Does this happen for `TerminateEndEvents`, or when you terminate a process manually?

Definitely, if I terminate it manually. With TerminationEndEvents I have not tested it
